content/reviewers_response.md

```diff
@@ -0,0 +1,632 @@
+
+```diff
+- There were a limited number of cases in which authors appeared to post preprints after the publication date, which results in preprints receiving a negative time difference, as previously reported [...]
+
++ We encountered 123 cases where the preprint posting date was subsequent to the publication date, resulting in a negative time difference, as previously reported [...].
+```
+
+> 12.) https://hypothes.is/a/DzrzGtLtEeuJlcdOlUfUBA
+> 
+> I'm surprised to see no citation given to JANE: https://jane.biosemantics.org/
+>
+>https://academic.oup.com/bioinformatics/article/24/5/727/202224
+>
+>reviewed in https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6300233/
+>
+>The 'find journals' functionality of JANE appears somewhat similar to the discovering similar journals functionality here.
+
+**Thank you for pointing this citation out. 
+We have incorporated it into our manuscript.**
+
+```diff
+- We developed a web application that places any bioRxiv or medRxiv preprint into the overall document landscape and identifies similar papers and journals.
+
++ We developed a web application that places any bioRxiv or medRxiv preprint into the overall document landscape and identifies topically similar papers and journals (similar to [..]).
+```
+
+> 13.) https://hypothes.is/a/eJ7-DtLpEeuh0A8OL8WAWw
+> 
+> This is the kind of journal-faff difference that I hypothesise would not be visible or less visible if one did an analysis of preprints vs author manuscripts.
+>
+>There is change but is change from figure to 'fig.' to suit journal style actually helpful/valuable? In my opinion it is not!
+
+**We recognize the limitation of our design choice and have mentioned using accepted author manuscripts as an alternative for future analyses.**
+
+> 14.) https://hypothes.is/a/-ixA_NLqEeuv6YviR-lI0A
+> 
+> I would cut this entire subsection from the manuscript to make it shorter (or relegate it to a supplementary section).
+>
+>Don't we already know that if one uses full texts we can determine the subfield of the paper? It's not that interesting in my opinion and not relevant to the main hypotheses of the paper -- comparing between preprints and the journal published version.
+
+**We believe this section is needed when performing a descriptive analysis on the bioRxiv corpus; however, we moved this section to the supplement.**
+
+> 15.) https://hypothes.is/a/V2wj3NLqEeuZ9idQ44yS4Q
+> 
+> adjusting for recency? i.e. not sampling 2019 preprints? in figure C the line indicates (if I'm interpreting correctly) that overall only 46.55% are published but that's because it includes very recent preprints that haven't had time to be journal published yet. Just be explicit that you are adjusting for recency (i.e. excluding 2019 and newer preprints) when you say that most preprints are eventually published.
+
+**We included all preprints (from 2013 to 2019/early 2020) posted onto biorxiv when calculating this proportion.**
```

Comment:
This response to reviewers is not precise. Can you be more precise about what you did? Is the manuscript updated to be precise:
> Just be explicit that you are adjusting for recency (i.e. excluding 2019 and newer preprints) when you say that most preprints are eventually published

If not, it should be. If yes, what change did you make? Put it here :)